### PR TITLE
Fix broken link to Google shell style guide

### PR DESCRIPTION
**Pull request by jonasc**

### CODE REVIEW

**Comment by Jcharly412** [comment]
Back up asap M1

**Comment by Jcharly412** [comment]
Transportation need information star 💫

**Comment by Jcharly412** [comment]
Stocks

**Comment by Jcharly412** [comment]
Mechanic

**Comment by Jcharly412** [comment]
Safety first

**Comment by Jcharly412** [comment]
Goals

**Comment by Jcharly412** [comment]
Respect responsibility

**Comment by Jcharly412** [comment]
Apple

**Comment by Jcharly412** [comment]
iPhone XR

**Comment by Jcharly412** [comment]
Family

**Comment by Jcharly412** [comment]
Wxyz

**Comment by Jcharly412** [comment]
Machine pro

**Comment by Jcharly412** [comment]
Over stocks red shelf

**Comment by Jcharly412** [comment]
Since day one

**Comment by Jcharly412** [comment]
💫🌎🤔❤️🎟⭐️📡🦅♐️🐐😘🍎🍏👩‍🚀🚆👰🏼‍♀️🧑‍🚀🧻🤴😎🪐🎷👨‍🚒☝️🛻🙏

**Comment by Jcharly412** [comment]
🦅ALL IN ONE 🌎 FOR A BETTER FUTURE 🤔

**Comment by Jcharly412** [comment]
♐️

**Comment by Jcharly412** [comment]
👨‍🏭

**Comment by Jcharly412** [comment]
🫡

**Comment by Jcharly412** [comment]
Ag

**Comment by Jcharly412** [comment]
Entertainment

**Comment by Jcharly412** [comment]
BIGO

**Comment by Jcharly412** [comment]
TikTok

**Comment by Jcharly412** [comment]
Influencers

**Comment by Jcharly412** [comment]
Since day one

**Comment by Jcharly412** [comment]
Chevron

**Comment by Jcharly412** [comment]
Microsoft

**Comment by Jcharly412** [comment]
Mac

**Comment by Jcharly412** [comment]
Intel

**Comment by Jcharly412** [comment]
IBM

**Comment by Jcharly412** [comment]
Hp

**Comment by Jcharly412** [comment]
John Deere

**Comment by Jcharly412** [comment]
Ag

**Comment by Jcharly412** [comment]
Agriculture

**Comment by Jcharly412** [comment]
Jose Carlos Cruz jr

**Comment by Jcharly412** [comment]
Intelligence at its best

**Comment by Jcharly412** [comment]
R1

**Comment by Jcharly412** [comment]
M1 M2 Connection since day one

**Comment by Jcharly412** [comment]
Platform

**Comment by Jcharly412** [comment]
Warp

**Comment by Jcharly412** [comment]
Verizon

**Comment by Jcharly412** [comment]
Starlink

**Comment by Jcharly412** [comment]
Wow

File: README.md [hunk 1]
```diff
@@ -826,7 +826,7 @@ performance of any of your sites from across the globe.<br>
 &nbsp;&nbsp;:small_orange_diamond: <a href="https://wiki.bash-hackers.org/start"><b>The Bash Hackers Wiki</b></a> - hold documentation of any kind about GNU Bash.<br>
 &nbsp;&nbsp;:small_orange_diamond: <a href="http://pubs.opengroup.org/onlinepubs/9699919799/utilities/contents.html"><b>Shell & Utilities</b></a> - describes the commands and utilities offered to application programs by POSIX-conformant systems.<br>
 &nbsp;&nbsp;:small_orange_diamond: <a href="https://github.com/jlevy/the-art-of-command-line"><b>the-art-of-command-line</b></a> - master the command line, in one page.<br>
-&nbsp;&nbsp;:small_orange_diamond: <a href="https://google.github.io/styleguide/shell.xml"><b>Shell Style Guide</b></a> - a shell style guide for Google-originated open-source projects.<br>
```
Comment (Trickskeee3013) [inline README.md:829]: This line seems a bit buggy though ? I bet it's nothing

**Review by Trickskeee3013 — COMMENTED** [review]
Hi and thank you for your time good sir

**Review by Trickskeee3013 — COMMENTED** [review]
- 829 please

**Pull request merged**